Description: rename shebang from node to nodejs
 See pkg-javascript policy.
Forwarded: not-needed
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2013-03-25
--- a/bin/node-gyp.js
+++ b/bin/node-gyp.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env nodejs
 
 /**
  * Set the title.
